### PR TITLE
ITS Online Calibration: solved data drops, updated input string to new format 

### PR DESCRIPTION
**Pull request by IsakovAD**
- Fixed data drops (by Ivan Ravasenga)
- New format of input string

### CODE REVIEW

File: Modules/ITS/src/ITSThresholdCalibrationTask.cxx [hunk 1]
```diff
@@ -214,7 +228,7 @@ ITSThresholdCalibrationTask::CalibrationResStruct ITSThresholdCalibrationTask::C
 {
   CalibrationResStruct result;
   auto StaveINFO = splitString(input, ",");
-
+  o2::itsmft::ChipMappingITS mp;
```
Comment (iravasen) [inline Modules/ITS/src/ITSThresholdCalibrationTask.cxx:231]: can you move this to the header file?

**Comment by IsakovAD** [comment]
@iravasen all comments were applied in the new commit

**Comment by iravasen** [comment]
Thanks @IsakovAD, approving (cc @knopers8 )

**Review by iravasen — APPROVED** [review]

**Comment by iravasen** [comment]
The error is not related to this PR. @Barthelemy or @knopers8 can you please force merge and then merge also https://github.com/AliceO2Group/ControlWorkflows/pull/509 which is directly linked to this PR? Thank you!

**Pull request merged**